src/Uno.UWP/UI/Core/ICoreWindowEvents.cs

```diff
@@ -16,5 +16,15 @@ public interface ICoreWindowEvents
 
 	void RaiseKeyUp(KeyEventArgs args);
 	void RaiseKeyDown(KeyEventArgs args);
+#if __SKIA__
+	// Due to the way keyboard events are handled in Skia,
+	// the CoreWindow is responsible for initiating the event
+	// bubbling flow. However, a RaiseKeyUp/Down event should
+	// be fired on CoreWindow itself AFTER the bubbling flow
+	// is done. So, we add another events as a workaround.
+
+	internal void RaiseNativeKeyDownReceived(KeyEventArgs args);
+	internal void RaiseNativeKeyUpReceived(KeyEventArgs args);
```

Comment:
Technically, this is a breaking change. But I'm also not aware of any "valid" scenario where consumers would implement this interface.